Backend/dev/test-tool/load-test-service/server.py

```
@@ -0,0 +1,114 @@
+
+        self._json({'error': 'Not found'}, 404)
+
+    def do_POST(self):
+        p = self.path.rstrip('/')
+        length = int(self.headers.get('Content-Length', 0))
+        raw = self.rfile.read(length) if length else b'{}'
+
+        if p == '/start':
+            try:
+                config = json.loads(raw)
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)